tests/e2e/Cart/PayPalButtonCart.block.spec.js

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+const { expect } = require('@playwright/test');
+const { test } = require('../Shared/base-test');
+const {setOrderAPI, markStatusInMollie} = require('../Shared/mollieUtils');
+const {addProductToCart} = require('../Shared/wooUtils');
+const {wooOrderPaidPage, wooOrderDetailsPageOnPaid} = require('../Shared/testMollieInWooPage');
+
+test.describe('PayPal Transaction in classic cart', () => {
+    test('Not be seen if not enabled', async ({page, products}) => {
+        await page.goto(process.env.E2E_URL_TESTSITE + '/wp-admin/admin.php?page=wc-settings&tab=checkout&section=mollie_wc_gateway_paypal');
```

Comment:
I think you can omit `process.env.E2E_URL_TESTSITE` since it was set in the config?

Also maybe extract our settings page URL somewhere.